### PR TITLE
Adding Klaytn, Dogechain, DFK, and Cronos

### PR DESCRIPTION
**Pull request by simonmahns**
## Description

Added Klaytn, Dogechain, DFK, and Cronos, ammended an error in the boba file.

### CODE REVIEW

File: packages/chains/src/cronos.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      name: 'Cronos',
+      symbol: 'CRO',
+    },
+    rpcUrls: {
+      default: { http: ['https://node.croswap.com/rpc'] },
+      public: { http: ['https://node.croswap.com/rpc'] },
+    },
+    blockExplorers: {
+      etherscan: { name: 'CronosScan', url: 'https://cronoscan.com/' },
+      default: { name: 'CronosScan', url: 'https://cronoscan.com/' },
```
Comment (tmm) [inline packages/chains/src/cronos.ts:18]:
```suggestion
      etherscan: { name: 'CronosScan', url: 'https://cronoscan.com' },
      default: { name: 'CronosScan', url: 'https://cronoscan.com' },
```

File: packages/chains/src/dfk.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Chain } from './types'
+
+export const dfk = {
+    id: 53935,
```
Comment (tmm) [inline packages/chains/src/dfk.ts:4]:
```suggestion
    id: 53_935,
```

File: packages/chains/src/dfk.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    name: 'DFK Chain',
+    network: 'dfk',
+    nativeCurrency: {
+      decimals: 18,
+      name: 'Jewel',
+      symbol: 'JEWEL',
+    },
+    rpcUrls: {
+      default: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc	'] },
+      public: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc	'] },
```
Comment (tmm) [inline packages/chains/src/dfk.ts:14]:
```suggestion
      default: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc'] },
      public: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc'] },
```

File: packages/chains/src/dogechain.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { Chain } from './types'
+
+export const dogechain = {
+    id: 2000,
```
Comment (tmm) [inline packages/chains/src/dogechain.ts:4]:
```suggestion
    id: 2_000,
```

File: packages/chains/src/dogechain.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    name: 'Dogechain',
+    network: 'dogechain',
+    nativeCurrency: {
+      decimals: 18,
+      name: 'Dogechain',
+      symbol: 'DC',
+    },
+    rpcUrls: {
+      default: { http: ['https://rpc.dogechain.dog	'] },
```
Comment (tmm) [inline packages/chains/src/dogechain.ts:13]:
```suggestion
      default: { http: ['https://rpc.dogechain.dog'] },
```

File: packages/chains/src/klaytn.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    name: 'Klaytn',
+    network: 'klaytn',
+    nativeCurrency: {
+      decimals: 18,
+      name: 'Klaytn',
+      symbol: 'KLAY',
+    },
+    rpcUrls: {
+      default: { http: ['https://cypress.fautor.app/archive	'] },
+      public: { http: ['https://cypress.fautor.app/archive	'] },
```
Comment (tmm) [inline packages/chains/src/klaytn.ts:14]:
```suggestion
      default: { http: ['https://cypress.fautor.app/archive'] },
      public: { http: ['https://cypress.fautor.app/archive'] },
```

File: packages/chains/src/klaytn.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { Chain } from './types'
+
+export const klaytn = {
+    id: 8217,
```
Comment (tmm) [inline packages/chains/src/klaytn.ts:4]:
```suggestion
    id: 8_217,
```

File: packages/chains/src/klaytn.ts [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+      name: 'Klaytn',
+      symbol: 'KLAY',
+    },
+    rpcUrls: {
+      default: { http: ['https://cypress.fautor.app/archive	'] },
+      public: { http: ['https://cypress.fautor.app/archive	'] },
+    },
+    blockExplorers: {
+      etherscan: { name: 'KlaytnScope', url: 'https://scope.klaytn.com/' },
+      default: { name: 'KlaytnScope', url: 'https://scope.klaytn.com/' },
```
Comment (tmm) [inline packages/chains/src/klaytn.ts:18]:
```suggestion
      etherscan: { name: 'KlaytnScope', url: 'https://scope.klaytn.com' },
      default: { name: 'KlaytnScope', url: 'https://scope.klaytn.com' },
```

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning)

**Comment by simonmahns** [comment]
> Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning)

Hey, added change set (first time using, cool software!) and fixed changes. Thanks for the feedback.

**Review by tmm — CHANGES_REQUESTED** [review]
Please make sure all these chains are exported from the `index.ts` file

**Pull request merged**